tests/python_package_test/test_basic.py

```diff
@@ -49,7 +49,7 @@ def test(self):
         for preds in zip(pred_from_matr, pred_from_model_file):
             self.assertEqual(*preds)
         # check pmml
-        os.system('python ../../pmml/pmml.py model.txt')
+        # os.system('python ../../pmml/pmml.py model.txt')
```

Comment:
Should we still keep pmml directory? I think https://github.com/jpmml/jpmml-lightgbm does a better job.